src/explore-education-statistics-common/src/components/form/rhf/__tests__/RHFFormFieldCheckboxSearchSubGroups.test.tsx

```diff
@@ -0,0 +1,971 @@
+import RHFFormFieldCheckboxSearchSubGroups from '@common/components/form/rhf/RHFFormFieldCheckboxSearchSubGroups';
+import RHFForm from '@common/components/form/rhf/RHFForm';
+import FormProvider from '@common/components/form/rhf/FormProvider';
+import Yup from '@common/validation/yup';
+import { waitFor } from '@testing-library/dom';
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import noop from 'lodash/noop';
+import React from 'react';
+
+describe('FormFieldCheckboxSearchSubGroups', () => {
+  test('renders with correct default ids without form', () => {
+    render(
+      <FormProvider
+        initialValues={{
+          test: [],
+        }}
+      >
+        {() => {
+          return (
```

Comment:
Similar to other test suites, could spare a few lines by not using the render prop variant to declare the children.